Cargo.toml

```diff
@@ -11,9 +11,23 @@ members = [
   "crates/holochain_env_setup",
 ]
 
+[workspace.dependencies]
+hpos-config-core = { git = "https://github.com/Holo-Host/hpos-config", rev = "a36f862869cc162c843ac27ed617910d68f480cc" }
+hpos-config-seed-bundle-explorer ={ git = "https://github.com/Holo-Host/hpos-config", rev = "a36f862869cc162c843ac27ed617910d68f480cc" }
+ed25519-dalek = "2.1.1"
+holochain_keystore = "0.4.0-dev.2"
+holochain_types = { version = "0.4.0-dev.2", features = ["chc"] }
+holochain_conductor_api = "0.4.0-dev.2"
+lair_keystore_api = "=0.4.4"
+holochain_websocket = "0.4.0-dev.2"
+holofuel_types = "0.5.9"
+sodoken = "0.0.11"
+serde = { version = "1.0", features = ["derive", "rc"] } # { version = "1.0.193", features = ["derive"] }
+mr_bundle = { version = "0.4.0-dev.1" }
+reqwest = { version = "0.12.4", features = ["json"]}
```

Comment:
Just a note: I think when you add this to the workspace. when you use one of these crates as a git dependency in other repos, it has a problem in building these crates. 


But if you're able to use these in other repos. then awesome

Reply:
I am able to use crates in other repos, e.g. hpos-api-rust